usecases/evaluate_scenario/evaluate_scenario.go

```diff
@@ -187,6 +189,24 @@ func EvalTestRunScenario(ctx context.Context,
 		),
 	)
 	defer span.End()
+	testrun, errTr := repositories.ScenarioTestRunRepository.GetTestRunByLiveVersionID(ctx, exec, *params.Scenario.LiveVersionID)
```

Comment:
IMO we can make things a bit simpler here (without looking up the scenario by its live version, as discussed there can be a situation where decisions are still being created on a previous version):
IF we receive a decision based on a version X
AND there is a test run in status "up" using "live version id" X
THEN we create a phantom decision on this test run
ELSE pass

Reply:
(that's an edge case though, broadly the logic is correct)

Reply:
I think this case is already handled. I just completed by testing the status of the testrun 

````golang
if testrun == nil || testrun.Status != models.Up
````